component/src/test/java/org/cbioportal/genome_nexus/util/GenomicVariantUtilTest.java

```diff
@@ -0,0 +1,68 @@
+package org.cbioportal.genome_nexus.util;
+
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class GenomicVariantUtilTest {
```

Comment:
Apparently we have test cases for `GenomicVariantUtil`, but in the wrong module with a wrong name... https://github.com/genome-nexus/genome-nexus/blob/b3bab9c9532e1641bbf0440bd992e98a291951ce/service/src/test/java/org/cbioportal/genome_nexus/util/GenomicVariantTest.java#L9

We should probably move those tests here as well, and remove the duplicate ones that I just added.

Reply:
👍